src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -71,5 +71,11 @@ def load_arguments(self, _):
         c.argument('container_name', help='The container name to tail the logs. If omitted, the first container in the container group will be chosen')
         c.argument('follow', help='Indicate to stream the tailing logs', action='store_true')
 
+    with self.argument_context('container exec') as c:
+        c.argument('container_name', help='The container name to exec the command. If omitted, the first container in the container group will be chosen')
+        c.argument('exec_command', help='The container to exec to.')
+        c.argument('terminal_row_size', help='The size for the terminal rows.')
+        c.argument('terminal_col_size', help='The size for the terminal columns.')
```

Comment:
What are these for?

Reply:
They are the size of the output for the command. I agree they should have much better help text. I will fix that now.